airbyte-integrations/connectors/source-snowflake/source_snowflake/schema_builder.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+
+
+class SchemaTypes:
+    string: Dict = {"type": ["null", "string"]}
+    number: Dict = {"type": ["null", "number"]}
+    integer: Dict = {"type": ["null", "number"]}
```

Comment:
Why not use the variable `number` in case of integers? It feels confusing for a user to see both but they are actually the same

Reply:
This remark is still valid regarding the mapping you have provided ? 
![image](https://github.com/artefactory/airbyte/assets/163012956/7831f449-3517-4b2e-b010-5c393e929b93)